PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 Before submitting, please confirm you've
  - [ ] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
- - [ ] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
+ - [ ] completed [Mattermost Contributor Agreement](https://mattermost.com/contribute/)
```

Comment:
This URL also goes to the Contribute page and not the Contributor Agreement - is this correct?